libs/repositories/src/datasources/cowApi.ts

```diff
@@ -3,7 +3,26 @@ import createClient from 'openapi-fetch';
 const COW_API_BASE_URL = process.env.COW_API_BASE_URL || 'https://api.cow.fi';
 
 import type { paths } from '../gen/cow/cow-api-types';
+import { ALL_CHAIN_IDS, SupportedChainId } from '../types';
 
-export const cowApiClientMainnet = createClient<paths>({
-  baseUrl: COW_API_BASE_URL + '/mainnet',
-});
+export type CowApiClient = ReturnType<typeof createClient<paths>>;
+
+/**
+ * Name of the networks as they are in the API
+ */
+const NETWORK_NAMES: Record<SupportedChainId, string> = {
```

Comment:
```suggestion
const COW_API_NETWORK_NAMES: Record<SupportedChainId, string> = {
```

Then the comment is not needed IMO.